### DISCUSSION TITLE
Failed to start the kernel

### DISCUSSION THREAD

**Discussion by gkarozis** [microsoft/vscode-jupyter#11994] [Questions and Answers] [answered]
Hello!

I am having trouble running jupyter notebook on vscode. When i try to run a cell of the notebook it tries to connect to the kernel:

Connecting to kernel: Python 3.10.8 64-bit: Activating Python Environment 'Python 3.10.8 64-bit'

Then I am having the error below:

Failed to start the Kernel. 
AttributeError: 'SelectIOLoop' object has no attribute 'asyncio_loop'. 

I was not having this problem a week before. I am running the code in Python 3.10.8. Thanks in advance

**Comment by DonJayamanne** [comment, marked as answer]
It looks like you have an old version of some packge, without the stack trace its difficult to tell which package needs updating.
you might want to try stackoverflow or creating a new environment
